web_console/dynamic-plugin/overview-dynamic-plugin.adoc

```diff
@@ -40,6 +42,12 @@ conster Header: React.FC = () => {
 
 * Avoid selectors that could affect markup outside of your plugins components, such as element selectors. These are not APIs and are subject to change. Using them might break your plugin. Avoid selectors like element selectors that could affect markup outside of your plugins components.
 * Provide valid JavaScript Multipurpose Internet Mail Extension (MIME) type using the `Content-Type` response header for all assets served by your plugin web server. Each plugin deployment should include a web server that hosts the generated assets of the given plugin.
+* You must build your plugin with Webpack using Webpack version 5+.
+* You can use link:https://nodejs.org/en/[`Node.js`] and link:https://yarnpkg.com/[`yarn`] to build and run your plugin.
```

Comment:
Picking nits here, but yarn is a package manager and doesn't actually do the build. npm (rather than node) is the alternative.

This is more a prereq like node and Docker I would say if we have those documented in another place.

Reply:
I removed it since we have as a prereq in some of the other docs.